doc/contribute/contributor_expectations.rst

```diff
@@ -93,6 +93,9 @@ PR Requirements
 - Each commit in the PR must provide a commit message following the
   :ref:`commit-guidelines`.
 
+- The PR must include a description that summarizes the changes in the PR
```

Comment:
nice,  but do you think people will read this? Still good to get in, but IMO we need to enforce this in CI, fail PRs without any description.

Reply:
Don't we do it already?

Reply:
Nevermind, PR description, not commit message -- yeah I agree ideally the CI would check this.

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/75788